plugins-pain-points.md

```diff
@@ -0,0 +1,85 @@
+# Sensu Community Plugins Pain Points
+
+## Testing
+
+- Some concern with determining the quality of plugins.
+  - Current focus on integration testing as first step.
+  - Regression testing across releases and platforms.
+  - Interest in performance testing
+- People loves when things work
+  - Imperative that CM or monitoring is not broken,
+  - Therefor for most plugin consumers it is important that they have the _confidence_ it works.
+
+## Performance
+
+-  Plugin installation can be very slow.
+  - Example: when using auto-scaling group and instances with 50 plugins by the time the new instance has converged the problem has fixed itself
+  - More and more plugins are using many / large dependencies
+  - Could be better documentation for performance concerns and solutions. 
+  - Possibly provide boilerplate code for embedding in debian package.
+  - Possibly provide boilerplate code for using GemInABox, etc.
+  - Boilerplate, boilerplate, and boilerplate...
+
+## Windows
+
+- Distinct lack of Windows plugins
+  - How can we empower the community to contribute more Windows plugins
+  - Majority of the current contributors and maintainers are not regularly administrating Windows environments
+  - Could use more documentation? More windows boilerplate? Scaffolding?
+
+## Contributing
+
+- Not immediately clear how to contribute new plugins.
```

Comment:
I am looking to address this in https://github.com/sensu-plugins/community/pull/42